it/it-tests/src/test/java/it/organization/OrganizationMembershipTest.java

```diff
@@ -152,57 +151,58 @@ public void user_creating_an_organization_becomes_member_of_this_organization()
   @Test
   public void should_display_members_page() {
     String orgKey = createOrganization();
-    userRule.createUser("foo", "pwd");
-    userRule.createUser("bar", "pwd");
+    String userFoo = createUser("foo");
+    String userBar = createUser("bar");
     createUser();
 
-    adminClient.organizations().addMember(orgKey, "foo");
-    adminClient.organizations().addMember(orgKey, "bar");
+    adminClient.organizations().addMember(orgKey, userFoo);
+    adminClient.organizations().addMember(orgKey, userBar);
 
     MembersPage page = nav.openOrganizationMembers(orgKey);
     page
       .canNotAddMember()
       .shouldHaveTotal(3);
     page.getMembersByIdx(0).shouldBeNamed("admin", "Administrator");
-    page.getMembersByIdx(1).shouldBeNamed("bar", "bar");
+    page.getMembersByIdx(1).shouldBeNamed(userBar, userBar);
     page.getMembersByIdx(2)
-      .shouldBeNamed("foo", "foo")
+      .shouldBeNamed(userFoo, userFoo)
       .shouldNotHaveActions();
   }
 
   @Test
   public void search_for_members() {
     String orgKey = createOrganization();
     String user1 = createUser();
-    String user2 = createUser();
-    createUser();
+    String user2 = createUser("sameprefixuser");
```

Comment:
🐞  If you want to increase the scope of this test, you could also use `util.user.UserRule#createUser(java.lang.String, java.lang.String, java.lang.String, java.lang.String)` to provide a name, different from the user's login.